keylime/policy/create_runtime_policy.py

```diff
@@ -304,12 +304,9 @@ def boot_aggregate_parse(line: str) -> Tuple[str, str]:
     :return: tuple with two values, the algorithm used and the digest of the
              boot aggregate
     """
-    def_alg = FALLBACK_HASH_ALGO
-    def_agg = "0" * algorithms.Hash(def_alg).hexdigest_len()
-
     alg, digest, fpath, ok = process_ima_sig_ima_ng_line(line)
     if not ok or fpath != "boot_aggregate":
-        return def_alg, def_agg
+        return "unknown", ""
```

Comment:
Any reason why to return "unknown" instead of something falsely?

Reply:
Or defining a constant or something similar, given it is used a few times throughout the code?

Reply:
@sergio-correia Yeah, it makes sense to make it a constant.

@THS-on When I was writing it, I felt the need to make it easy to tell apart the case when we tried to guess the algorithm and failed from the case when there was no input.

And I already see some issues coming from the fact that `sha256` and `sm3_256` has the same output length. I will probably address this in a future PR.

If you agree, I'll follow Sergio's suggestion and make it a constant. If not, I'll try to figure a way without using it.

Reply:
Yeah let's make it a constant